static/styles/compose.css

```diff
@@ -89,6 +95,10 @@
     flex-flow: column;
 
     .stream-selection-header-colorblock {
+        box-shadow: none;
+        border: 1px solid hsl(0, 0%, 80%);
+        border-right: none;
```

Comment:
Changed from box-shadow to border to match the stream name, since the discrepancy was more obvious after the background change